torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10454,7 +10455,8 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                    supports_sparse_csc=True,
                    supports_sparse_bsr=True,
                    supports_sparse_bsc=True,
-                   supports_forward_ad=True),
+                   supports_forward_ad=True,
+                   enable_skipped_device=('xpu',)),
```

Comment:
Is it possible to provide a utility function dedicated to xpu to enable some particular operations to be tested on xpu? Otherwise, we have to add more codes with more operations being enabled. And when XPU op coverage is good enough, the code is useless.

Reply:
Added enable_skipped_op_dict in common_utils.py to read the xpu_op_db.yaml file. Added enable_skipped_device in OpInfo to skip the unsupported op.